pkg/venafi/tpp/connector_test.go

```diff
@@ -439,7 +440,10 @@ func TestRequestCertificateServiceGenerated(t *testing.T) {
 	req.CsrOrigin = certificate.ServiceGeneratedCSR
 	req.FetchPrivateKey = true
 	req.KeyPassword = "newPassw0rd!"
-
+	req.CustomFields = []certificate.CustomField{
+		certificate.CustomField{Type: certificate.CustomFieldPlain, Name: "TestField", Value: "Value 1"},
+		certificate.CustomField{Type: certificate.CustomFieldPlain, Name: "TestField2", Value: "TRUE"},
+	}
```

Comment:
We already test custom fields in TestRequestAndSearchCertificate

Reply:
I will look at that test to make sure it will test this logic. If so, I will remove these tests so we don't have multiple custom field requirements